### PR TITLE
update based on recent changes to tekton and knative

### PR DESCRIPTION
**Pull request by beemarie**

### CODE REVIEW

File: workshop/exercise-7/README.md [hunk 1]
```diff
@@ -95,42 +94,41 @@ A `PipelineResource` is used to define artifacts that are passed into and out of
...
+            - --context=$(params.pathToContext)
     ```
 
 2. Apply this yaml file to your kubernetes cluster.
     ```
     kubectl apply -f image-from-source-task.yaml
     ```
   
-3. Now that our `Task` is defined, let's define a `TaskRun` that will run the `Task`. This `TaskRun` will bind inputs and outputs to our earlier defined resources, `fib-knative` and `fib-knative-git`. This `TaskRun` will also execute the `steps` we defined for our `Task`. Take a look at this file, called `image-from-source-task-run.yaml`.
+3. Now that our `Task` is defined, let's define a `TaskRun` that will run the `Task`. This `TaskRun` will bind inputs and outputs to our earlier defined resources, `fib-knative` and `fib-knative-git`. This `TaskRun` will also execute the `steps` we defined for our `Task`. We also define a `serviceAccount` for this `TaskRun` to utilize for any required permissions. Take a look at this file, called `image-from-source-task-run.yaml`.
```
Comment (duglin) [inline workshop/exercise-7/README.md:107]: I wonder if we need to say a little bit more about ServiceAccounts?  I don't want to dive too deep, but this one sentence might be a bit too little.

File: workshop/exercise-7/README.md [hunk 2]
```diff
@@ -185,10 +183,11 @@ A `PipelineResource` is used to define artifacts that are passed into and out of
     Expected Output:
 
       ```
-      NAME                                            READY   STATUS      RESTARTS   AGE
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   3/3     Running     0          56s
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   3/3   Terminating   0          89s
-      fib-knative-00002-deployment-58dcbdb97c-rrnzc   0/3   Terminating   0          91s
+      NAME                                                    READY   STATUS      RESTARTS   AGE
+      build-docker-image-from-git-source-task-run-pod-lwhl2   0/4     Completed   0          18h
+      fib-knative-built-rjn2g-deployment-6f6d9766cb-mr57f     2/2     Running     0          12s
```
Comment (duglin) [inline workshop/exercise-7/README.md:188]: just wondering... are the last 2 columns really not aligned when you run the cmd?

File: workshop/exercise-7/README.md (hunk 1 above)
Comment (beemarie) [reply workshop/exercise-7/README.md:107]: we give a little explanation the first time service accounts are mentioned, in exercise 6:
```
5. A `Service Account` provides an identity for processes that run in a Pod. This Service Account will be used to link the build process for Knative to the Secrets you just created. View the service account file, and notice that it's using the credentials you created earlier for pulling from and pushing to the container registry:
```

File: workshop/exercise-7/README.md (hunk 2 above)
Comment (beemarie) [reply workshop/exercise-7/README.md:188]: yeah, this is actually how it looks.

Comment (beemarie) [reply workshop/exercise-7/README.md:188]: image

**Pull request merged**